shell/browser/hid/electron_hid_delegate.cc

```diff
@@ -76,6 +76,13 @@ bool ElectronHidDelegate::HasDevicePermission(
                                               render_frame_host);
 }
 
+void ElectronHidDelegate::RevokeDevicePermission(
+    content::RenderFrameHost* render_frame_host,
+    const device::mojom::HidDeviceInfo& device) {
+  // TODO(jkleinsc) implement this for
+  // https://chromium-review.googlesource.com/c/chromium/src/+/3297868
```

Comment:
Should we have a tracking bug for this?

Reply:
- I'm already working on this, but I did create an issue here to track: #32842